Library/Homebrew/cmd/audit.rb

```diff
@@ -512,9 +512,9 @@ def audit_github_repository
     end
 
     problem "GitHub fork (not canonical repository)" if metadata["fork"]
-    if (metadata["forks_count"] < 10) && (metadata["subscribers_count"] < 10) &&
-       (metadata["stargazers_count"] < 20)
-      problem "GitHub repository not notable enough (<10 forks, <10 watchers and <20 stars)"
+    if (metadata["forks_count"] < 50) && (metadata["subscribers_count"] < 50) &&
+       (metadata["stargazers_count"] < 100)
+      problem "GitHub repository not notable enough (<50 forks, <50 watchers and <100 stars)"
```

Comment:
Maybe we could remove the numbers but keep an explanation, something like “not enough forks/watchers/stars” to be clearer than just “not notable enough”. That would prevent the transparency concern @cytopia is talking about in their comment below.